tests/package_tests/internals/common.py

```diff
@@ -252,17 +252,20 @@ def perform(
                 if bytes_sent <= 0:
                     return (
                         LogVerifierCheckResult.FAIL,
-                        "Agent log says that during the run the agent has sent zero bytes.",
+                        f"Agent log says that during the run the agent has sent zero bytes.\n"
+                        f"Whole log content: {whole_log_text}",
```

Comment:
It seems that some changes have not been merged yet from the master, but there has to be the similar output of the whole test on the outer level https://github.com/scalyr/scalyr-agent-2/blob/79e7066a73b868121677068e276cbe6a8a597342/tests/package_tests/internals/common.py#L189-L200

There may be a double output, I guess

Reply:
I will test the change on failure and see how it behaves and remove this change if it's not needed anymore (I added it since in the pat it was hard to troubleshoot failures since they were missing the context).